### PR TITLE
refactor: "small" refactor job

### PR DESCRIPTION
**Pull request by tarlepp**

### CODE REVIEW

File: Command/CheckCommand.php [hunk 1]
```diff
@@ -27,8 +28,18 @@
...
+ * Class CheckCommand
+ *
+ * @package Protacon\Bundle\TestToolsBundle\Command
+ */
 class CheckCommand extends Command
 {
+    /**
+     * @var string The default command name
+     */
+    static public $defaultName = 'test-tools:check';
```
Comment (tarvainen) [inline Command/CheckCommand.php]: public static

File: Command/InitCommand.php [hunk 2]
```diff
@@ -24,6 +26,11 @@
  */
 class InitCommand extends Command
 {
+    /**
+     * @var string The default command name
+     */
+    static public $defaultName = 'test-tools:check';
```
Comment (tarvainen) [inline Command/InitCommand.php]: public static

File: Util/ComposerManager.php [hunk 3]
```diff
@@ -1,11 +1,18 @@
 <?php
+declare(strict_types = 1);
 
 namespace Protacon\Bundle\TestToolsBundle\Util;
 
+use RuntimeException;
```
Comment (tarvainen) [inline Util/ComposerManager.php]: Should also use file_put_contents and file_exists?

Comment (tarlepp) [reply Util/ComposerManager.php]: This should be fixed now.

**Pull request merged**